chia/rpc/wallet_rpc_api.py

```diff
@@ -1244,21 +1244,28 @@ async def nft_get_current_nfts(self, request):
         return {"wallet_id": wallet_id, "success": True, "nfts": nft_uri_pairs}
 
     async def nft_transfer_nft(self, request):
+        assert self.service.wallet_state_manager is not None
         wallet_id = int(request["wallet_id"])
+        new_did_inner_hash = request.get("new_did_inner_hash", "")
+        trade_price = request.get("trade_price", [[0]])
         nft_wallet: NFTWallet = self.service.wallet_state_manager.wallets[wallet_id]
         # nft_coin_info: NFTCoinInfo,
         # new_did,
         # new_did_parent,
         # new_did_inner_hash,
         # new_did_amount,
         # trade_price,
-        sb = await nft_wallet.transfer_nft(
-            request["nft_coin_info"],
-            bytes.fromhex(request["new_did"]),
-            request["new_did_inner_hash"],
-            request["trade_price"],
-        )
-        return {"wallet_id": wallet_id, "success": True, "spend_bundle": sb}
+        try:
+            sb = await nft_wallet.transfer_nft(
+                request["nft_coin_info"],
+                bytes.fromhex(request["new_did"]),
+                new_did_inner_hash,
+                trade_price,
+            )
+            return {"wallet_id": wallet_id, "success": True, "spend_bundle": sb}
+        except Exception as e:
+            log.error(f"Failed to transfer NFT: {e}")
```

Comment:
might be useful to use `log.exception` here to show full stack trace in logs for easier debugging